src/exoscale/tools/project/io.clj

```diff
@@ -9,7 +9,7 @@
 
 (defn shell
   "Run shell commands contained in `cmds`. The first unsuccesful exit triggers a system exit."
-  [cmds {:keys [dir env out]}]
+  [cmds {:keys [dir env out fatal?] :or {fatal? true}}]
```

Comment:
I wouldn't make it the default, we have cases where we want it to sys.exit 1

Reply:
I set `fatal?` default to `true` in order to not introduce any breaking change as we used to always `sys/exit` when a shell command failed
Anything I missed or misunderstood ?

Reply:
nevermind, I somehow read `fatal? false`. Looks good to me.